### ISSUE TITLE
Testing mode for Holler Box

### ISSUE THREAD

**Issue by josephsellers** [josephsellers/rcp-hollerbox#3]
This plugin allows us to target users in WP using RCP controls. We set it up so that if a user dismisses the Holler Box it won't show again even on a different device. The dismissal gets stored in their usermeta. 

However, we are finding it super difficult to test new Holler Boxes. If we dismiss the box while testing we can never get it to show up again. Even using User Switch plugin and changing users still won't get it to show. 

Can we add a 'Testing Mode' or a button to clear out all dismissals for a particular box?

Basically, it works for our customers once we have it up and running, it is just very difficult to test.

**Comment by volodymyr-tsaryk** [comment]
@josephsellers you also need to clear cookie while switching user
More simple would be to allow clear HB submission data for the particular user(delete date from meta and clear cookies automatically)

**Comment by josephsellers** [comment]
@volodymyr-tsaryk yeah, we tried clearing cookies (all site data from the browser) but maybe it gets re-set immediately? Because we never manage to clear it and see the Holler Box. I have tried many workarounds and clearing cookies/site data but can never find a reliable way to show these boxes.

**Comment by volodymyr-tsaryk** [comment]
Yes, if meta was there cookie was set immediately after page reload.
So I will put the estimate for adding a button that will clear custom HB user meta and cookies

**Comment by volodymyr-tsaryk** [comment]
I did it more flexible way, you just need to clear value of this text area http://take.ms/EpW0N and save user profile, and if it value will be empty it will clear cookies and set user meta

**Comment by volodymyr-tsaryk** [comment]
I did it more flexible way, you just need to clear value of this text area http://take.ms/EpW0N and save user profile, and if it value will be empty it will clear cookies and set user meta

**Comment by josephsellers** [comment]
Works great, thanks!
